compiler/rustc_middle/src/dep_graph/mod.rs

```diff
@@ -85,6 +85,10 @@ impl rustc_query_system::dep_graph::DepKind for DepKind {
     fn can_reconstruct_query_key(&self) -> bool {
         DepKind::can_reconstruct_query_key(self)
     }
+
+    fn is_anon(&self) -> bool {
```

Comment:
```suggestion
    #[inline]
    fn is_anon(&self) -> bool {
```

`can_reconstruct_query_key` also seems to be missing it.